src/app/legacy/containers/PageHandlers/withOptimizelyProvider/index.jsx

```diff
@@ -28,7 +28,7 @@ const withOptimizelyProvider = Component => {
   return props => {
     const { service } = useContext(ServiceContext);
     const isStoryBook = process.env.STORYBOOK;
-    const disableOptimizely = isStoryBook;
+    const disableOptimizely = isStoryBook || isCypress;
```

Comment:
I've just realised that when running the cypress tests the "old" way, the optimizely stuff was stubbed: https://github.com/bbc/simorgh/blob/3a9e6e0b33f55c4525de2adef99eada0499d16f4/cypress/support/helpers/runTestsForPage.js#L44-L57

Do we need to implement the same logic in https://github.com/bbc/simorgh/blob/36e3e79761d42822919d76550c2c72cf199915c9/ws-nextjs-app/cypress/support/helpers/runTestsForPage.ts?

(We should also keep this fix)

Reply:
I think those tests will still work ok, this is just more to disable Optimizely experiments running in Cypress, not so much disable the library itself. The JSON file and all will still be downloaded in Cypress (which is fine).

Reply:
Just trying to wrangle this failing test which is considerably more difficult than it looks (or I just don't know how to fix it 😓 ).

We're setting Cypress globally in Jest: https://github.com/bbc/simorgh/blob/1a1d835d3399afced408a209720c4a5e42d6d628/src/testHelpers/jest-setup.js#L26-L28 which is interfering with this new `isCypress` check as its always returning an object, so its truthy.

Overriding globals in Jest seems particularly difficult for some reason.

Reply:
I am trying to remember what the purpose of those lines were. It might have been so that we would catch if visiting page failed when optimizely had that status code or an invalid object? As we would not want any optimizely problems to cause the pages to error.